// [START declaration]
syntax = "proto3";
package files;
// [END declaration]

// [START go_declaration]
option go_package = "./filespb";
// [END go_declaration]


// [START messages]
// [START ratings messages]
message RatingCSV {
  int32 userId = 1;
  int32 movieId = 2;
  float rating = 3;
  string timestamp = 4;
}

// Batch of ratings from file
message RatingsCSV {
  repeated RatingCSV ratings = 1;
}
// [END ratings messages]

// [START credits messages]
message CreditCSV {
  string cast = 1;
  string crew = 2;
  int32 id = 3;
}

// Batch of credits from file
message CreditsCSV {
  repeated CreditCSV credits = 1;
}
// [START credits messages]
// [END messages]